testnets/archwaytestnet/assetlist.json

```diff
@@ -53,6 +53,188 @@
+          }
+        }
+      ]
+    },
+    {
+      "description": "Circle's USDC Noble Testnet (Grand) on Archway Testnet",
+      "denom_units": [
+        {
+          "denom": "ibc/CE6B3F114B3B6EAEC14E9221A7E799B16D394C9D5D7CF644156F481A9E69EC5D",
+          "exponent": 0,
+          "aliases": ["uusdc"]
+        },
+        {
+          "denom": "usdc",
+          "exponent": 6
+        }
+      ],
+      "type_asset": "ics20",
+      "base": "ibc/CE6B3F114B3B6EAEC14E9221A7E799B16D394C9D5D7CF644156F481A9E69EC5D",
+      "name": "USDC (Noble)",
+      "display": "usdc",
+      "symbol": "USDC",
+      "traces": [
+        {
+          "type": "synthetic",
+          "counterparty": {
+            "chain_name": "forex",
+            "base_denom": "USD"
+          },
+          "provider": "Circle"
+        },
+        {
+          "type": "bridge",
+          "counterparty": {
+            "chain_name": "ethereum",
+            "base_denom": "0xa0b86991c6218b36c1d19d4a2e9eb0ce3606eb48"
+          },
+          "provider": "Circle"
+        },
+        {
+          "type": "ibc",
+          "counterparty": {
+            "chain_name": "noble",
+            "base_denom": "uusdc",
+            "channel_id": "channel-215"
+          },
+          "chain": {
+            "channel_id": "channel-500",
+            "path": "transfer/channel-500/uusdc"
+          }
+        }
+      ],
+      "logo_URIs": {
+        "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/noble/images/USDCoin.png",
+        "svg": "https://raw.githubusercontent.com/cosmos/chain-registry/master/noble/images/USDCoin.svg"
+      },
+      "images": [
+        {
+          "image_sync": {
+            "chain_name": "noble",
+            "base_denom": "uusdc"
+          },
+          "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/noble/images/USDCoin.png",
+          "svg": "https://raw.githubusercontent.com/cosmos/chain-registry/master/noble/images/USDCoin.svg",
+          "theme": {
+            "primary_color_hex": "#2474cc"
+          }
+        }
+      ]
+    },
+    {
+      "description": "Wrapped USDC (Noble Testnet Grand)",
+      "type_asset": "cw20",
+      "address": "archway1r8kepegwhldwqanuurc769l2g0qxlsm2sm6t5rhqjzcerxsgshls267f7a",
+      "denom_units": [
+        {
+          "denom": "cw20:archway1r8kepegwhldwqanuurc769l2g0qxlsm2sm6t5rhqjzcerxsgshls267f7a",
+          "exponent": 0
+        },
+        {
+          "denom": "wUSDC",
+          "exponent": 6
+        }
+      ],
+      "base": "cw20:archway1r8kepegwhldwqanuurc769l2g0qxlsm2sm6t5rhqjzcerxsgshls267f7a",
+      "name": "Wrapped USDC",
+      "display": "wUSDC",
+      "symbol": "wUSDC",
+      "traces": [
+        {
+          "type": "synthetic",
+          "counterparty": {
+            "chain_name": "forex",
+            "base_denom": "USD"
+          },
+          "provider": "Circle"
+        },
+        {
+          "type": "bridge",
```

Comment:
I suspect the bridged is incorrect here. I recommend keeping only the latest trace hop registered for each asset.